### PR TITLE
Update for new Timer API

### PR DESCRIPTION
**Pull request by mantognini**
Adapt stainless to changes introduced in epfl-lara/inox#39. Doesn't compile as-is: `build.sbt` needs to be updated and refer to a newest version of inox.

`TerminationComponent` will be updated after #67 is merged (merge conflict anyway).

### CODE REVIEW

File: core/src/main/scala/stainless/verification/VerificationChecker.scala [hunk 1]
```diff
@@ -131,10 +131,11 @@ trait VerificationChecker { self =>
         }
       } catch {
         case u: Unsupported =>
-          val t = timer.selfTimer.get
-          val time = if (t.isRunning) t.stop else t.runs.last
+          val time = if (timer.isRunning) timer.stop else timer.time
```
Comment (samarion) [inline core/src/main/scala/stainless/verification/VerificationChecker.scala]: Why don't we use the new API here?

Comment (mantognini) [reply core/src/main/scala/stainless/verification/VerificationChecker.scala]: The `run` facility unfortunately doesn't give access to the elapsed time and we need it here for the `VCResult`.

Comment (samarion) [reply core/src/main/scala/stainless/verification/VerificationChecker.scala]: Ah I see, that's too bad. This is fine then.

Comment (mantognini) [reply core/src/main/scala/stainless/verification/VerificationChecker.scala]: TBH I hesitated to introduce some `tryRun[T](b: => T): (Long, Try[T])` in the `TimerStorage` API. We could still do it, although its usage would be limited to 1 or 2 places. What do you think?

Comment (samarion) [reply core/src/main/scala/stainless/verification/VerificationChecker.scala]: As you say, the use cases are so rare (for now) that I don't think there's any need for such a method. If in the future we find out we need it more, we can rethink this.

**Comment by samarion** [comment]
I published a new inox version 1.0.2-191-g3b196ab with the latest timer API version.

**Comment by mantognini** [comment]
Thanks, it seems the integration suite went fine!

**Comment by mantognini** [comment]
Unless some one has any concern I'll merge this.

**Comment by samarion** [comment]
I thought you were waiting on https://github.com/epfl-lara/stainless/pull/67?

**Comment by mantognini** [comment]
It's not a big deal, but I'd rather merge this one first because #76 depends on #67, *and* requires the newest inox version too.

**Comment by samarion** [comment]
Ok, sounds good. Just don't forget the termination component :)

**Comment by mantognini** [comment]
Done in https://github.com/epfl-lara/stainless/pull/67/commits/32b0843687f6ccdce9e33e1b89f4fd2f42ce1ad5 ;-)

**Pull request merged**